tasks/setup.yaml

```diff
@@ -5,25 +5,31 @@ tasks:
         description: The version of k3d-core-slim-dev to deploy
         # renovate: datasource=github-tags depName=defenseunicorns/uds-core versioning=semver
         default: 0.24.1
+      snapshots:
+        description: Whether to pull from the nightly snapshot repository
+        default: "false"
       insecure_keycloak_admin:
         description: Automatically set a keycloak admin username / password
         default: "true"
     actions:
       - description: Create k3d cluster with slim UDS Core
-        cmd: ./uds deploy oci://defenseunicorns/uds/bundles/k3d-core-slim-dev:${{ .inputs.version }} --set INSECURE_ADMIN_PASSWORD_GENERATION=${{ .inputs.insecure_keycloak_admin }} --confirm --no-progress
+        cmd: ./uds deploy oci://defenseunicorns/uds/${{ if eq .inputs.snapshots "true" }}snapshots/${{ end }}bundles/k3d-core-slim-dev:${{ .inputs.version }} --set INSECURE_ADMIN_PASSWORD_GENERATION=${{ .inputs.insecure_keycloak_admin }} --confirm --no-progress
```

Comment:
I think this is okay, although we maybe could also filter on `version` = some regex that matches date string + commit sha to auto-detect if its a snapshot or not (or just check valid semver vs not)? Then wouldn't need someone to pass in snapshot explicitly, just a version?

Reply:
IMO that feels clunky - I think this is simple enough (and unlikely to be used a ton - though I may eat my words there) - gonna merge as is and we can change it later